### PR TITLE
Xatolar to'g'rilandi

### PR DESCRIPTION
**Pull request by mrIminjonov**

### CODE REVIEW

File: TestSchoolWebApi/Models/Dtos/AddressRequestDto.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿namespace TestSchool.Models.Dtos
+{
+    public class AddressRequestDto
+    {
+        public string AddressId { get; set; }
```
Comment (isfandiyoro5) [inline TestSchoolWebApi/Models/Dtos/AddressRequestDto.cs]: AdressId kerak emas

File: TestSchoolWebApi/Models/Dtos/AddressRequestDto.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+        public string Country { get; set; }
+
+        public string City { get; set; }
+
+        public string Region { get; set; }
```
Comment (isfandiyoro5) [inline TestSchoolWebApi/Models/Dtos/AddressRequestDto.cs]: Region Adress class da yoq lekin sen Dtoga qoshib qoyibsan

File: TestSchoolWebApi/Models/Dtos/AddressResponseDto.cs [hunk 3]
```diff
@@ -0,0 +1,11 @@
+﻿namespace TestSchool.Models.Dtos
+{
+    public class AddressResponseDto
+    {
+        public string Addressid { get; set; }
+
+        public string Country { get; set; }
+
+        public string City { get; set; }
```
Comment (isfandiyoro5) [inline TestSchoolWebApi/Models/Dtos/AddressResponseDto.cs:9]: Street ni ham qoshish kk

**Review by isfandiyoro5 — CHANGES_REQUESTED** [review]

File: TestSchoolWebApi/Models/Dtos/AddressRequestDto.cs (hunk 1 above)
Comment (mrIminjonov) [reply TestSchoolWebApi/Models/Dtos/AddressRequestDto.cs]: ok togrilemiz

File: TestSchoolWebApi/Models/Dtos/AddressResponseDto.cs (hunk 3 above)
Comment (mrIminjonov) [reply TestSchoolWebApi/Models/Dtos/AddressResponseDto.cs:9]: responseda streetni korsatishni istamadim sababi malumotlar ni deyarli hammasi ochiq bolib qoladi unda request bilan responseni farqi qomedi

**Review by JmlKhon — APPROVED** [review]

**Review by isfandiyoro5 — APPROVED** [review]

**Pull request merged**